qt/python/mantidqt/project/recovery/projectrecovery.py

```diff
@@ -0,0 +1,471 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantidqt package
+#
+
+from __future__ import (absolute_import, unicode_literals)
+
+import datetime
+import getpass
+import os
+import psutil
+import shutil
+import time
+from glob import glob
+from qtpy.QtCore import QMetaObject, Q_ARG, Qt
```

Comment:
Can we section the imports as defined in the [standards](http://developer.mantidproject.org/Standards/PythonStandards.html)? `threading` is part of the standard library.

Reply:
PyCharm organise imports does this, so I suggest running that through

Reply:
PyCharm Organise imports produced the state of affairs we have:

![image](https://user-images.githubusercontent.com/24723609/53332709-ea3f5580-38eb-11e9-8960-27c7d1c6d2b3.png)

Whilst I believe our standards want something like:
![image](https://user-images.githubusercontent.com/24723609/53332735-f9be9e80-38eb-11e9-9235-a0ec62221592.png)

Reply:
Interesting, mine is correctly sorted, and it seems PyCharm follows the PEP8 [style guide](https://www.python.org/dev/peps/pep-0008/#imports), so I'm not sure why there would be a difference, maybe a PyCharm version difference? Mine is 2018.3.1

![image](https://user-images.githubusercontent.com/9135965/53335486-7d2fbe00-38f3-11e9-8ccf-82524b4512ce.png)

Reply:
A lot of these imports will be changing due to a pending refactor, so unfortunately this is kind of negligible.

However I have:
![image](https://user-images.githubusercontent.com/24723609/53339086-3777f300-38fd-11e9-9277-4eb4088edaca.png)